daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -37,7 +37,9 @@ var (
 	// We retry device removal so many a times that even error messages
 	// will fill up console during normal operation. So only log Fatal
 	// messages by default.
-	DMLogLevel int = devicemapper.LogLevelFatal
+	DMLogLevel                   int  = devicemapper.LogLevelFatal
+	DriverDeferredRemovalSupport bool = false
+	EnableDeferredRemoval        bool = false
```

Comment:
Seems like a good idea to make this true by default? Like, this should be the expected behavior... but I guess we don't want to change people's envs too much.